src/GraphQL.Tests/Execution/Performance/ThreadPerformanceTests.cs

```diff
@@ -1,16 +1,17 @@
 using System.Diagnostics;
+using GraphQL.DI;
 using GraphQL.Types;
 using GraphQL.Utilities;
 
 namespace GraphQL.Tests.Execution.Performance
 {
     public class ThreadPerformanceTests : QueryTestBase<ThreadPerformanceTests.ThreadPerformanceSchema>
     {
-        public ThreadPerformanceTests()
+        public override void RegisterServices(IServiceRegister Services)
         {
             Services.Register<PerfQuery>();
-
-            Services.Singleton(() => new ThreadPerformanceSchema(new SimpleContainerAdapter(Services)));
+            Services.Register<PerfMutation>();
```

Comment:
In some tests (3 or 4), some registrations were lost. It worked because `SimpleContainer` created an unregistered services under the hood.

Reply:
I'm glad that the registrations are now explicit.